sktime/forecasting/ardl.py

```diff
@@ -243,6 +246,23 @@ def __init__(
         self.fixed = fixed
         self.causal = causal
         self.trend = trend
+        # Safe optional dependency check for 'ctt'
+        if self.trend == "ctt":
+            try:
+                import statsmodels
```

Comment:
The presence of `statsmodels` is already checked via the `python_dependencies` tag.

So I think this part can be removed. The check for sm > 0.15.0 I would route via `_check_soft_dependencies` with `severity="none"`, that is cached and minimizes boilerplate overheads.